README.md

```diff
@@ -19,56 +19,64 @@ The applause button is a custom element that can be added directly to the page.
 </head>
 <body>
   <!-- add the button! -->
-  <applause-button style="width: 58px; height: 58px;"/>
+  <applause-button style="width: 4rem" />
 </body>
 ```
 
 The above will render an applause button, persisting claps and clap counts. 
 
 For more information, visit the [project website](https://colineberhardt.github.io/applause-button/);
 
+## Browser Support
+
+All browsers with
+[partial support of Custom Elements](https://caniuse.com/custom-elementsv1); check current state at
+[Browserslist](https://browsersl.ist/#q=partially+supports+custom-elementsv1).
+
 ## Development
 
 ![Node LTS](https://img.shields.io/node/v-lts/applause-button?logo=nodedotjs)
 
-Clone this repo and install dependencies via yarn:
+Clone this repo then install dependencies via yarn:
```

Comment:
worth mentioning we use yarn berry?

Reply:
Actually we're using Yarn Classic - see `.yarnrc` and the `packageManager` property in package.json. This should be compatible with all installed versions of Yarn.

I would actually prefer to switch to npm, as some things are tricker with yarn (such as running a dependency audit). But I'll open a separate issue for that.